integration/flink/app/src/test/resources/events/expected_cassandra.json

```diff
@@ -5,11 +5,11 @@
     "name": "flink_examples_cassandra"
   },
   "inputs": [{
-    "namespace": "flink",
-    "name": "source_event"
+    "namespace": "${json-unit.any-string}",
```

Comment:
What's the value of this in a test? Can we retrieve records from mockServer and verify programatically in a test class?

Reply:
It is the value for my local run. cassandra://172.20.0.8:9042

Reply:
I changed to a concrete value for integration. I think it is more readable of hostname exist in endpoint.